views/home.njk

```diff
@@ -125,7 +125,7 @@
       </div>
       <div class="feature-text">
         <h2>Fully customizable</h2>
-        <p>Build and deploy custom sites that take advantage of Federalist's scalable static hosting architecture.</p>
+        <p>Build and deploy custom sites that take advantage of Federalist's scalable static hosting architecture. Federalst hosts 90 diverse federal sites at present; contact us to learn more about what you can do with Federalist.</p>
```

Comment:
I would suggest making the "contact us" text a link to federalist-inquiries@gsa.gov or linking that text to the Google form.